gensim/utils.py

```diff
@@ -1146,15 +1146,15 @@ def keep_vocab_item(word, count, min_count, trim_rule=None):
         else:
             return default_res
 
-def check_output(*popenargs, stdout=subprocess.PIPE, **kwargs):
+def check_output(stdout=subprocess.PIPE, *popenargs, **kwargs):
```

Comment:
this keeps default `stdout=subprocess.PIPE`. and check_output calls in all wrappers are updated to specify cmd as keyword argument rather than positional to work with this change